test/IISTestSite/Startup.cs

```diff
@@ -325,5 +331,81 @@ private void CheckRemoveAuthEnvironmentVariable(IApplicationBuilder app)
                 await context.Response.WriteAsync(variable);
             });
         }
+        private void ReadAndWriteSynchronously(IApplicationBuilder app)
+        {
+            app.Run(async context =>
+            {
+                var t2 = Task.Run(() => WriteManyTimesToResponseBody(context));
+                var t1 = Task.Run(() => ReadRequestBody(context));
+                await Task.WhenAll(t1, t2);
+            });
+        }
+
+        private async Task ReadRequestBody(HttpContext context)
```

Comment:
Add a test that does CopyToAsync from the request to the response body.

Reply:
Done.